package.json

```diff
@@ -14,18 +14,21 @@
 		"autoprefixer": "^10.4.14",
 		"eslint": "8.35.0",
 		"eslint-config-next": "13.2.3",
+		"howler": "^2.2.3",
```

Comment:
It seems that you installed `howler` but don't use it in your code as you decided to use `use-sound` instead.
Please remove the unused library.

Reply:
yes exactly, I had experimented with hovler before, but did not get the result I had wished for. Is uninstalled! :-)